FordLabs/retroquest
# Move end retro logic to backend

## Overview
Presently all of the logic to end a retro exists only in the frontend. This should be handled in the backend not only to simplify the frontend logic and limit the number of independant database save calls made, but also so that anyone using the RetroQuest API can also end a retro.

## Testing Instructions
How to test this PR
* Create a new team
* Sign into the team in two browsers
* Add thoughts and action items
* Mark some thoughts and action items complete
* Archive retro
* The only things left on the board are uncompleted action items
* There should now be a board with the current date under the thoughts archive tab
* There should now be your completed action items under the action items archive tab
